tests/testGalSimInterface.py

```diff
@@ -1107,6 +1111,112 @@ def test_checkpointing(self):
                                  gs_img.wcs.fitsHeader.get(name))
 
 
+class GetStampBoundsTestCase(unittest.TestCase):
+    """
+    TestCase class for the GalSimInterpreter.getStampBounds
+    function.
+    """
+    def setUp(self):
+        self.scratch_dir = tempfile.mkdtemp(dir=ROOT, prefix='GetStampBounds')
+        self.db_name = os.path.join(self.scratch_dir, 'galsim_test_db')
+
+    def tearDown(self):
+        try:
```

Comment:
Do you mind replacing these with
```
if os.path.exists(self.db_name):
    os.unlink(self.db_name)
if os.path.exists(self.scratch_dir):
    os.rmdir(self.scratch_dir)
```
That way, if we have trouble with multithreading or NFS mounted systems, we will know.